src/Plugin.php

```diff
@@ -356,7 +356,7 @@ public static function loadTextdomain() {
    */
   public static function paginate_links($link) {
     if (is_paged()) {
-      $link = preg_replace('@/page/1/?@', user_trailingslashit(''), $link);
+      $link = preg_replace('@/page/1/?(\?|$)@', user_trailingslashit(''), $link);
```

Comment:
The captured delimiter of the query string needs to be reinjected. See https://regex101.com/r/xUKfaM/2
```suggestion
      $link = preg_replace('@/page/1/?(\?|$)@', user_trailingslashit('') . '$1', $link);
```